src/axolotl/prompt_tokenizers.py

```diff
@@ -85,7 +85,11 @@ def _tokenize(self, prompt: str, add_eos_token=True, strip_bos_token=False):
             result["input_ids"].append(self.tokenizer.eos_token_id)
             result["attention_mask"].append(1)
 
-        if result["input_ids"][0] == self.tokenizer.bos_token_id and strip_bos_token:
+        if (
+            len(result["input_ids"]) > 0
```

Comment:
I guess if we wanted to reuse variables, we can optionally extract this to a variable and reuse the variable instead. This PR is good enough though.